.github/ISSUE_TEMPLATE/plant-epic.md

```diff
@@ -0,0 +1,60 @@
+---
+name: 📋 Plant Epic
+about: Overview of plant specific tasks
+title: "[Plant Epic]: "
+labels: epic
+assignees: ''
+
+---
+
+<!--
+This is a proposed minimum (!) list of plant related tasks. Feel free to add more tasks relevant for the specific plant or to customize. But focus on an overview in this epic, i.e. do not go too much into details.
+Each bullet point can easily be changed to an issue and described in more detail there.
+-->
+
+### Detailed Engineering
+- [ ] Select hardware
+- [ ] Create network plan
```

Comment:
```suggestion
- [ ] Create network plan
- [ ] Select hardware
```
We are running into trouble (redundancy tasks, steag/SENS fiber optics, ...)  a lot as we don't have a network plan when ordering hardware.